Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx

```diff
@@ -396,14 +395,14 @@ void GRPDCSDPsProcessor::updateEnvVarsCCDB()
 void GRPDCSDPsProcessor::updateCollimatorsCCDB()
 {
 
-  // we need to update a CCDB for the Env Variables DPs --> let's prepare the CCDBInfo
+  // we need to update a CCDB for the Collimators Variables DPs --> let's prepare the CCDBInfo
 
   if (mVerbose) {
     LOG(info) << "Entry related to Collimators needs to be updated with startTime " << mStartValidityColli;
   }
   std::map<std::string, std::string> md;
   md["responsible"] = "Chiara Zampolli";
-  o2::calibration::Utils::prepareCCDBobjectInfo(mEnvVars, mccdbCollimatorsInfo, "GLO/Config/Collimators", md, mStartValidityColli, mStartValidityColli + 3 * o2::ccdb::CcdbObjectInfo::DAY); // valid for 3 days
+  o2::calibration::Utils::prepareCCDBobjectInfo(mCollimators, mccdbCollimatorsInfo, "GLO/Config/Collimators", md, mStartValidityColli, mStartValidityColli + 3 * o2::ccdb::CcdbObjectInfo::DAY); // valid for 3 days
```

Comment:
Ops... I guess nobody ever used this CCDB... So hopefully it is not too bad. Weird that I did not notice, though, I think that when I tested locally, I checked all CCDB entries I was created. Anyway, thanks for fixing it!

Reply:
prepareCCDBobjectInfo uses the object just to fetch some metadata, which is anyway not used, the object itself is stored correctly.

Reply:
Ah ok! good then!